lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -265,7 +265,12 @@ def create_enrollment(self, expired):
                 user=self.user,
                 course_id=course.id
             )
-            ScheduleFactory(start=self.THREE_YEARS_AGO + datetime.timedelta(days=1), enrollment=enrollment)
+            ScheduleFactory(
+                # TODO remove 'start' field in removing writes from old field step in column renaming release
+                start=self.THREE_YEARS_AGO + datetime.timedelta(days=1),
+                start_date=self.THREE_YEARS_AGO + datetime.timedelta(days=1),
+                enrollment=enrollment
+            )
```

Comment:
This PR should really be a branch off of the 2/5 PR branch - so that only the 3/5 differences are shown here.